### PR TITLE
minor cleanup, create docker volume for rstudio

### PR DESCRIPTION
**Pull request by MatthewBemis**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/resources/jupyter/rstudio-docker-compose.yaml [hunk 1]
```diff
@@ -11,5 +11,10 @@ services:
...
       OWNER_EMAIL: "${OWNER_EMAIL}"
+    volumes:
+      - rstudio-working:/home/rstudio
     env_file:
-    - /etc/google_application_credentials.env
\ No newline at end of file
+      - /etc/google_application_credentials.env
+
+volumes:
+  rstudio-working:
```
Comment (vkumra-broad) [inline src/main/resources/jupyter/rstudio-docker-compose.yaml]: nit: new line

File: src/main/resources/reference.conf [hunk 2]
```diff
@@ -61,7 +61,7 @@ liquibase {
 # Jupyter proxy server configuration
 proxy {
   jupyterProxyDockerImage = "broadinstitute/openidc-proxy:2.3.1_2"
-  proxyServerName = "jupyter-proxy-server"
+  proxyServerName = "proxy-server"
```
Comment (vkumra-broad) [inline src/main/resources/reference.conf]: Is this change related to docker volume change?

Comment (MatthewBemis) [reply src/main/resources/reference.conf]: no

**Review by vkumra-broad — APPROVED** [review]
Would be good to split unrelated changes into multiple PRs in the future.

**Comment by MatthewBemis** [comment]
@vkumra-broad these 3 changes should have been in https://github.com/DataBiosphere/leonardo/pull/718 so in that sense I would consider them related. But yes, in general I agree with you.
